### PR TITLE
Feat/stytch

### PR DESCRIPTION
**Pull request by BurntVal**
Super rough stytch implementation. Mostly to get other eyes on it

https://github.com/burnt-labs/cosmos-kit/assets/117188229/00e2b62b-42a9-4317-8e38-3a1daaf6843c

### CODE REVIEW

File: packages/react/src/modal/modal.tsx [hunk 1]
```diff
@@ -44,11 +44,21 @@ export function WalletModal({
 
   const current = walletRepo?.current;
 
+  const [stytchState, setStytchState] = useState<State>(
+    // @ts-ignore
```
Comment (BurntNerve) [inline packages/react/src/modal/modal.tsx:48]: Why the ts-ignore?

File: wallets/stytch/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (BurntNerve) [inline wallets/stytch/.editorconfig:1]: Does this file need to be here? Or is it just for meeting patterns?

**Review by OjiCode — COMMENTED** [review]
Two questions

File: packages/react/src/modal/modal.tsx (hunk 1 above)
Comment (BurntVal) [reply packages/react/src/modal/modal.tsx:48]: It for some reason doesn't register the stytchState field as part of the client object. This is temporary. Ideally we can get it solved before actually pushing

File: wallets/stytch/.editorconfig (hunk 2 above)
Comment (BurntVal) [reply wallets/stytch/.editorconfig:1]: Yea just meeting patterns. In order to add your own wallet they recommend just copying the "leap" wallet and making adjustments so this is a file that just got dragged along. Also the reason for the 31k lines

**Comment by BurntVal** [comment]
UPDATE: Implementation is flawed. I am awaiting UI inputs on a class that should have 0 dependencies on UI. What I mean is that someone can use the `react-lite` and they would be infinitely suspended because the modal doesn't exist, so they never actually connect.

**Comment by justinbarry** [comment]
Work moved [here](https://github.com/burnt-labs/abstraxion). Will integrate with CosmoKit at some point in the future.

**Pull request closed without merging**